src/containers/Tenant/Query/QueryEditorControls/QueryEditorControls.tsx

```diff
@@ -1,5 +1,7 @@
 import block from 'bem-cn-lite';
 
+import {LabelWithPopover} from '../../../../components/LabelWithPopover';
```

Comment:
Import order. First we import external packages. Then internal packages (`gravity-ui`, `ydb-platform`). Then in-project imports, from the farthest to the nearest